src/game.zig

```diff
@@ -403,11 +405,26 @@ pub const DamageType = enum(u8) {
 };
 
 pub const Player = struct { // MARK: Player
+	pub const EyeData = struct {
+		pos: Vec3d,
+		vel: Vec3d,
+		coyote: f64,
+		step: @Vector(3, bool),
+		box: collision.Box,
+		desiredPos: Vec3d,
+	};
 	pub var super: main.server.Entity = .{};
-	pub var eyePos: Vec3d = .{0, 0, 0};
-	pub var eyeVel: Vec3d = .{0, 0, 0};
-	pub var eyeCoyote: f64 = 0;
-	pub var eyeStep: @Vector(3, bool) = .{false, false, false};
+	pub var eyeData: EyeData = .{
```

Comment:
Please move refactorings like this into a separate PR, see also the contributing guidelines on PR size.